app-tracking-protection/vpn-impl/build.gradle

```diff
@@ -91,6 +91,7 @@ dependencies {
 
     implementation KotlinX.coroutines.core
 
+    testImplementation project(':vpn-store-test')
```

Comment:
Added one fake that is in its own test gradle module. Like we do for `:common-test`